docs/pages/guides/using-push-notifications-services.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Using Push Notifications
+description: Learn about different push notification services.
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+
+Expo apps can work with any notification service or capabilities offered by the operating system. The following services provide purpose-built Expo integrations for implementing push notifications in your app:
+
+> The [`expo-notifications`](/versions/latest/sdk/notifications/) library is designed and tested to work with Expo's push notification service. Some advanced features may not be compatible with third-party providers, as they often have their own native and React Native SDKs optimized for their services.
+
+## Expo push notifications
+
+[Expo Notifications](/versions/latest/sdk/notifications/) provides a unified API for handling push notifications across Android and iOS. It integrates seamlessly with your Expo account and is free to use.
+
+### Key capabilities
+
+- Fully compatible with the [`expo-notifications`](/versions/latest/sdk/notifications/) library
+- Includes an Expo Dashboard to track notifications and user engagement
+- Supports testing notifications with the [Expo Notifications Tool](https://expo.dev/notifications)
+
+### Considerations and limitations
+
+- Notification extensions are not built-in, but you can set them up manually following Apple's guidelines.
+- Limited to 600 notifications per second per project, which works for most apps but may not suit high-volume scenarios
+
+For implementation details, see the following guides:
+
+<BoxLink
+  title="Expo push notifications overview"
+  description="Learn more about Expo push notifications."
+  href="/push-notifications/overview"
```

Comment:
```suggestion
  href="/push-notifications/overview"
  Icon={NotificationBoxIcon}
```

Let's import the icon from `import { NotificationBoxIcon } from '@expo/styleguide-icons/outline/NotificationBoxIcon';` since this what we use in Expo dashboard.